### ISSUE TITLE
Docker build fails

### ISSUE THREAD

**Issue by milovanderlinden** [Influenzanet/authentication-service#8]
Branch: dev
Go version: 1.14

Dockerfile assumes packages.txt, but the file is not available. I also notice that there is a Gopkg .toml present. Are you using dep? This should be reflected in the readme

**Comment by phev8** [comment]
Yes, Dockerfile is outdated here - I created a builder container that can run the dep tasks successfully: phev8/go-dep-builder

packages.txt is removed - was a workaround temporary back when it was not working properly with dep - but since then figured it out.
